### ISSUE TITLE
Graphics freeze after full minimize when returning to ShadPS4 window but the Game still runs in background with sound and controls

### ISSUE THREAD

**Issue by Virgula0** [shadps4-emu/shadPS4#1629]
Used version v0.4.1 (latest pre-release)
ALT-Tabbing while gaming and coming back to the shadps4 window works fine and lets the game be playable but if the game gets fully minimized when coming back the graphics seem to be frozen even if you can hear and control the character of the game

## Proof

https://youtu.be/GlFkEulZvsw

## System settings

- GPU: RTX 3060 Ti 8Gb VRAM; Driver Version: 566.1.4
- CPU: Intel i5 12600K
- RAM: 32Gb ddr4 3200Mhz
- OS: Windows 11 23H2

- Full screen option enabled ✅

**Comment by Virgula0** [comment]
The problem is present even without full screen option enabled

**Comment by Hermiten** [comment]
Is it still revelant ?

**Comment by Virgula0** [comment]
> Is it still revelant ?

What do you mean with "is it still relevant"?

**Comment by polybiusproxy** [comment]
They're asking if the issue is still happening.

**Comment by Virgula0** [comment]
I'll recheck as soon as possible

**Comment by xamionex** [comment]
Present for me, If you alt-tab and do something else it's fine, but as soon as you minimize the game window (pressing WIN+D is the fastest way to check) the game will freeze and later error out saying:
[Debug] <Critical> vk_swapchain.cpp:operator():79: Assertion Failed!
Failed to create swapchain: ErrorInitializationFailed

Also want to clarify, I'm playing bloodborne using mainBB from diego,
@ me if you believe it won't happen on stable and I'll test it too

**Comment by xamionex** [comment]
Still happens on latest prerelease cef92fbcaad34e0caa90bc5afbce5e5b30c43636

Game crashes with:
[Debug] <Critical> vk_swapchain.cpp:operator():81: Assertion Failed!
Failed to create swapchain: ErrorInitializationFailed

**Comment by Virgula0** [comment]
In few days I should be able to recheck this

**Comment by xamionex** [comment]
> In few days I should be able to recheck this

I don't think you need to load fully into games, you could just sit on main menu and hit Win+D
(or whatever's the equivalent to show desktop on your linux de)

It always crashes when the window gets minimized, not sure if this happens on linux tho, havent tested
Alt-tabbing is fine and even non-focusing is fine too, just minimizing is what causes it

Also to clarify my crashes are with BloodBorne (cusa03173) on win10, and yes this happens on every build of shadps4 so far

**Comment by Virgula0** [comment]
Tested on latest main version `0.5.1 windows-qt` the bug seems to be gone. If anyone else can confirm this it would be top. @xamionex @Hermiten

**Comment by xamionex** [comment]
Okay so upon further testing it seems to be fixed
However, when launching shadps4 via steam as a non steam shortcut is still crashes like before

~~if using an overlay of sorts, for example I have the shortcut for the game set up as a non-steam game which adds steam overlay to the game, when doing win+d when launching via steam it still crashes like before~~
~~RTSS doesn't seem to affect it~~
~~Doesn't seem to happen with discord overlay as well~~

Update: It's not steam input nor steam overlay, I've disabled both and it still crashes when starting from steam, not sure what to test or try

**Comment by Virgula0** [comment]
> Okay so upon further testing it seems to be fixed
> However, when launching shadps4 via steam as a non steam shortcut is still crashes like before
> 
> ~~if using an overlay of sorts, for example I have the shortcut for the game set up as a non-steam game which adds steam overlay to the game, when doing win+d when launching via steam it still crashes like before~~
> ~~RTSS doesn't seem to affect it~~
> ~~Doesn't seem to happen with discord overlay as well~~
> 
> Update: It's not steam input nor steam overlay, I've disabled both and it still crashes when starting from steam, not sure what to test or try

Are you still referring to the original problem or something else? The bug was the graphic freeze on window minimization I don't know about steam version or any other crash mentioned.

**Comment by xamionex** [comment]
> Are you still referring to the original problem or something else? The bug was the graphic freeze on window minimization I don't know about steam version or any other crash mentioned.

It's the same issue but only happens when launching with steam, do you want me to open anothr issue?

**Comment by Hermiten** [comment]
I'm closing this one. Please have a look if one exist for steam or open a new one if not. 
Thanks you
